usaspending_api/database_scripts/tableview_generator/tableview_sql_generator.py

```diff
@@ -0,0 +1,417 @@
+"""
+A TableView is
```

Comment:
incomplete comment?

Reply:
This module does a _lot_ ... so some description of what is intended and not intended to be in here would be helpful.

Reply:
... oops ...